### PR TITLE
fix h36 enqueue operation to accept policies for prospective year

### PR DESCRIPTION
**Pull request by saipraveen18**
https://www.pivotaltracker.com/story/show/186215618

### CODE REVIEW

File: app/event_source/subscribers/fdsh/insurance_policies/posted_subscriber.rb [hunk 1]
```diff
@@ -54,17 +57,22 @@ def process_insurance_policies_posted_event_for_h36(subscriber_logger, response,
...
+                     message = enqueue.success
+                     subscriber_logger.info "on_h36_insurance_policies_posted acked #{message.is_a?(Hash) ? message[:event] : message}"
+                     message
+                   else
+                     failure = error_messages(enqueue)
+                     subscriber_logger.info "process_h36_insurance_policies_posted_event: failure: #{failure}"
+                     failure
+                   end
           subscriber_logger.info "process_h36_insurance_policies_posted_event: ------- end"
+          result
```
Comment (saikumar9) [inline app/event_source/subscribers/fdsh/insurance_policies/posted_subscriber.rb:70]: @ saipraveen18 We don't need to return the result from this method as it is not being used. So you can remove lines 70, 67, and 63 and remove the assignment in line 60.

File: spec/event_source/subscribers/fdsh/insurance_policies/posted_subscriber_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+require 'shared_examples/family_response3'
+
+RSpec.describe Subscribers::Fdsh::InsurancePolicies::PostedSubscriber, dbclean: :after_each do
+  include_context "family response"
+
+  let(:logger) { instance_double("::Logger", info: nil, debug: nil, warn: nil, error: nil) }
+
+  # Please refactor this test once we move to logic for multiple assistance years into an operation
+  describe 'process_insurance_policies_posted_event_for_h36' do
+    context "prospective assistance_year" do
```
Comment (saikumar9) [inline spec/event_source/subscribers/fdsh/insurance_policies/posted_subscriber_spec.rb:13]: Also, can you update the tests to verify the creation of the transaction(for the given policy) for the expected transmission for the cases where the insurance policy is for the past year, current year, and prospective year?

**Review by saikumar9 — CHANGES_REQUESTED** [review]

**Review by saikumar9 — APPROVED** [review]
lgtm!

**Review by vkghub — APPROVED** [review]
LGTM

**Pull request merged**